pep-0639.rst

```diff
@@ -1,63 +1,66 @@
 PEP: 639
 Title: Improving License Clarity with Better Package Metadata
-Version: $Revision$
-Last-Modified: $Date$
-Author: Philippe Ombredanne <pombredanne at nexb.com>,
-        C.A.M. Gerlach <CAM.Gerlach at Gerlach.CAM>
-Sponsor: Paul Moore <p.f.moore at gmail.com>
-PEP-Delegate: Brett Cannon <brett at python.org>
+Author: Philippe Ombredanne <pombredanne@nexb.com>,
+        C.A.M. Gerlach <CAM.Gerlach@Gerlach.CAM>,
+Sponsor: Paul Moore <p.f.moore@gmail.com>
+PEP-Delegate: Brett Cannon <brett@python.org>
 Discussions-To: https://discuss.python.org/t/12622
 Status: Draft
 Type: Standards Track
 Content-Type: text/x-rst
 Created: 15-Aug-2019
-Post-History: 15-Aug-2019, 17-Dec-2021
+Post-History: `15-Aug-2019 <https://discuss.python.org/t/2154>`__,
+              `17-Dec-2021 <https://discuss.python.org/t/12622>`__,
```

Comment:
<img width="326" alt="image" src="https://user-images.githubusercontent.com/1324225/163722697-4537d91a-267e-4272-b314-35f00f452cab.png">

Do we need this final comma, or will some other tooling/CSS remove from display it later?


```suggestion
              `17-Dec-2021 <https://discuss.python.org/t/12622>`__
```

Reply:
Great catch; while the linting checks in #2484 explicitly allow (but of course not require) trailing commas (for the same reasons as in Python source), I'd somehow forgotten that the build system wasn't actually automatically trimming them yet in the output. I've implemented that as part of #2467 , which is blocked on #2484 , which would suggest merging those first...but wait, the plot thickens.

As part of that, looking through the code I actually happened to notice a significant problem with allowing manual email obfuscation, which makes the automatic obfuscation not work and actually offers substantially _less_ protection, which I discuss in #2484 (and affects this PEP, which this PR fixes). As such, that either requires a pretty complex code fix (which would take some thought and effort before its ready), or reverting part of the reversion to allow naive manual email obfuscation in the source (which would in turn require either some of the changes in this PEP, or conflicting with it). So we have a PR reference cycle, heh.

To fix this for now, I fixuped the initial commit to not add the trailing comma, as well as one other minor fix, so this should be ready now.